src/lib/flexbox/api/base.ts

```diff
@@ -65,10 +91,17 @@ export abstract class BaseFxDirective implements OnDestroy {
   // *********************************************
 
   /**
+   * Abstract method...
```

Comment:
Thanks for drawing my attention to this. This was only a partial (in fact poor) solution.
I found another more robust approach.